tox.ini

```diff
@@ -45,9 +45,10 @@ deps = flake8>=3
 commands = python -m flake8
 
 [testenv:coverage]
-skip_install = true
 deps = coverage
-commands = python -m coverage run run-tests.py --nosub
+commands = 
+     python -m coverage run run-tests.py --nosub
```

Comment:
Sorry, very minor comment: there are no reasons to have both `python -m coverage` and `coverage`. The `python -m` syntax was aimed at Windows, but since this is only ran on Ubuntu, just `coverage` is fine.
```yaml
commands = 
     coverage run run-tests.py --nosub
     coverage xml
```